docs/input_cheatsheet.md

```diff
@@ -0,0 +1,29 @@
+# Input Cheatsheet
```

Comment:
I dont know if it's a good idea to add and maintain this page. Everything should be documented in our Go struct definition (avoiding duplicates and a maintainability cost here)